_data/lifecycle-phases.yml

```
@@ -0,0 +1,161 @@
+          The USWDS core team determined that the component could be a good fit for the design system.
+          Components in this phase are ready for development but have not yet been assigned.
+        starts_when: The USWDS core team approves the component proposal.
+        ends_when: The component is assigned for development.
+        contribute_by:
+          - Volunteering to take the assignment and develop the component
+        find_more: Find all approved components on the discussion board.
+        reach_out: <a href="mailto:uswds@gsa.gov">Reach out to the USWDS core team</a> if you are interested in building an approved component.
+      - name: Conditionally approved
+        description: |
+          The USWDS core team determined that the component could be a good fit for the design system, but other work must be completed before development can be scheduled.
+          Development for these components will be put on hold until the related work is finished.
+          Once the related work is complete, the component will move to the “approved” state.
+        starts_when: The USWDS core team approves the component proposal.
+        ends_when: The dependent work is complete and the component moves to approved status.
+        find_more: Find all conditionally approved components on the discussion board.
+      - name: Returned for revision
+        description: |
+          The USWDS core team determined that the proposal for the component has not not yet made the case for inclusion, but could be approved with revisions and improvements.
+        starts_when: |
+          The USWDS core team returns a proposal for revision.
+        ends_when: |
+          Contributors to the component discussion address the gaps in the proposal, and the USWDS core team updates the proposal for re-evaluation.
+        contribute_by:
+          - Adding missing information to the discussion.
+        find_more: You can find all proposals returned for revision on the discussion board.
+      - name: Will not pursue
+        description: |
+          The core team determined that the proposed component is not a good fit for the design system.
+          We will neither assign nor develop these components for USWDS.
+        find_more: You can find a list of  “will not pursue” components on the discussion board.
+  - name: Development
+    description: Components undergoing active design, development, testing, and documentation before public release.
+    subphases:
+      - name: Assigned
+        description: |
+          A team committed to building this component.
+          The assignee might be the USWDS core team or a team from the USWDS community.
+          Only approved components are available for assignment.
+        starts_when: The component is assigned to a team for development.
+        ends_when: Component development begins.
+        find_more: You can find a list of all assigned components on the discussion board.
```

Need to add link once it is available.
```suggestion
        find_more: You can find a list of all [assigned components]() on the discussion board.
```